run_dir/design/running_notes_tab.html

```diff
@@ -6,11 +6,25 @@
                 <div class="panel-heading">Add New Running Note</div>
                 <div class="panel-body">
                     <div class="row">
-                        <div class="col-md-6">
+                        <div class="col-md-7 form-inline">
+                          <label>Choose category:</label>
+                          <div class="btn-toolbar rn-cat-filter" data-toggle="buttons" style="margin-top:5px">
```

Comment:
I think the class name `rn-cat-filter` is a bit confusing since it's not a filter anymore. But I can live with that.

Reply:
I think this is my fault, sorry 👀